index.html

```diff
@@ -1034,13 +1038,19 @@ <h2 id="BaseAudioContext">
               <li>In case of failure, queue a task on the <a>control thread</a>
               to execute the following, and abort these steps
                 <ol>
+                  <li>Reject all promises from <a>pendingResumePromises</a> in
+                  order, then clear <a>pendingResumePromises</a>.
+                  </li>
                   <li>Reject <em>promise</em>
```

Comment:
If you create an AudioContext and call `resume()` when audio is not allowed, the promise will be left pending until you call `resume()` while audio is allowed. Then, the two promises will be resolved.